WordPress/WordPressTest/NotificationContentRouterTests.swift

```diff
@@ -2,26 +2,12 @@
 import XCTest
 @testable import WordPress
 
-class NotificationContentRouterTests: XCTestCase {
-
-    let utility = NotificationUtility()
-    var sut: NotificationContentRouter!
-    var coordinator: MockContentCoordinator!
-
-    override func setUp() {
-        super.setUp()
-        utility.setUp()
-        coordinator = MockContentCoordinator()
-    }
-
-    override func tearDown() {
-        utility.tearDown()
-        super.tearDown()
-    }
-
+class NotificationContentRouterTests: CoreDataTestCase {
```

Comment:
Nitpick: Adding a new line here would keep the formatting consistent with the majority of other tests

```suggestion
class NotificationContentRouterTests: CoreDataTestCase {

```

Thinking about it, I wonder why we don't have a Hound / SwiftLint warning for this? 🤔 

Part of the upcoming core work should definitely be ensuring we make the most out fo SwiftLint.

Reply:
I'm going to commit this to avoid a back and forth between the two of us. It's only a formatting change after all.

Reply:
I was going to commit your suggestion. You beat me to it. 😄